client/src/screens/MyProfile/index.js

```diff
@@ -9,6 +9,7 @@ import { FlexRow } from '../../components/Flex';
 import { buildYearlyTotals } from './buildYearlyTotals';
 import { theme } from '../../shared/theme';
 import { formatCurr } from '../../helpers/formatCurr';
+import SubscribeUser from '../../components/SubscribeUser/index';
```

Comment:
can this just be `import SubscribeUser from '../../components/SubscribeUser';`

Reply:
Not sure how that happened